### DISCUSSION TITLE
Use @rspack/plugin-html or html-webpack-plugin?

### DISCUSSION THREAD

**Discussion by chenjiahan** [web-infra-dev/rspack#4393] [Q&A] [answered]
Rspack [v0.3.4](https://github.com/web-infra-dev/rspack/releases/tag/v0.3.4) has provided support for `html-webpack-plugin`, I would like to know if using `@rspack/plugin-html` is still recommended, or it will deprecated in the future?

@hardfist @ahabhgk

**Comment by hardfist** [comment, marked as answer]
I think @rspack/plugin-html will be deprecated in the future
